apps/meteor/client/views/marketplace/AppMenu.js

```diff
@@ -140,7 +144,7 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 		const confirm = async () => {
 			closeModal();
 			try {
-				const { status } = await setAppStatus({ status: 'manually_disabled' });
+				const { status } = await setAppStatus({ status: 'manually_disabled', version: app.version });
```

Comment:
The field `version` shouldn't be sent anymore to the endpoint - remove all usages